src/hotspot/cpu/aarch64/frame_aarch64.cpp

```diff
@@ -1,6 +1,7 @@
 /*
  * Copyright (c) 1997, 2021, Oracle and/or its affiliates. All rights reserved.
  * Copyright (c) 2014, 2021, Red Hat Inc. All rights reserved.
+ * Copyrithg (c) 2025, Datadog, Inc. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2025, Datadog, Inc. All rights reserved.
```

I tend to agree that this doesn't really need a copyright update.

Reply:
Ok. Removed. 
Usually I was asked to add a copyright line so I was pro-active this time :)
But I have no strong opinion about this.